apidef/oas/schema/x-tyk-gateway.md

```diff
@@ -254,92 +301,55 @@ DomainToCertificates maintains the mapping of domain to certificate.
 DomainToPublicKeysMapping maintains the mapping of domain to pinned public keys.
 
-Tyk classic API definition: `pinned_public_keys`.
-
-
-### **PinnedPublicKey**
-
-**Field: `domain` (`string`)**
-Domain contains the domain name.
-
-**Field: `publicKeys` (`[]string`)**
-PublicKeys contains a list of the public keys pinned to the domain name.
-
-
-### **Server**
-
-**Field: `listenPath` ([ListenPath](#listenpath))**
-ListenPath represents the path to listen on. Any requests coming into the host, on the port that Tyk is configured to run on, that match this path will have the rules defined in the API definition applied.
-
-**Field: `slug` (`string`)**
-Slug is the Tyk Cloud equivalent of listen path.
-
-Tyk classic API definition: `slug`.
-
-**Field: `authentication` ([Authentication](#authentication))**
-Authentication contains the configurations related to authentication to the API.
-
-**Field: `clientCertificates` ([ClientCertificates](#clientcertificates))**
-ClientCertificates contains the configurations related to static mTLS.
-
-**Field: `gatewayTags` ([GatewayTags](#gatewaytags))**
-GatewayTags contains segment tags to configure which GWs your APIs connect to.
-
-**Field: `customDomain` ([Domain](#domain))**
-CustomDomain is the domain to bind this API to.
 
-Tyk classic API definition: `domain`.
-
-**Field: `detailedActivityLogs` ([DetailedActivityLogs](#detailedactivitylogs))**
-DetailedActivityLogs configures detailed analytics recording.
+Tyk classic API definition: `pinned_public_keys`.
 
+# ListenPath
 
-### **ListenPath**
+ListenPath represents the path the server should listen on.
 
 **Field: `value` (`string`)**
 Value is the value of the listen path e.g. `/api/` or `/` or `/httpbin/`.
 
 Tyk classic API definition: `proxy.listen_path`.
 
 **Field: `strip` (`boolean`)**
-Strip removes the inbound listen path in the outgoing request. e.g. `http://acme.com/httpbin/get` where `httpbin`is the listen path. The `httpbin` listen path which is used to identify the API loaded in Tyk is removed, and the outbound request would be `http://httpbin.org/get`.
+Strip removes the inbound listen path in the outgoing request. e.g. `http://acme.com/httpbin/get` where `httpbin`
+is the listen path. The `httpbin` listen path which is used to identify the API loaded in Tyk is removed,
+and the outbound request would be `http://httpbin.org/get`.
 
 Tyk classic API definition: `proxy.strip_listen_path`.
 
+# Authentication
 
-### **Authentication**
+Authentication types contains configuration about the authentication methods and security policies applied to requests.
 
 **Field: `enabled` (`boolean`)**
 Enabled makes the API protected when one of the authentication modes is enabled.
 
+
 Tyk classic API definition: `!use_keyless`.
 
 **Field: `stripAuthorizationData` (`boolean`)**
 StripAuthorizationData ensures that any security tokens used for accessing APIs are stripped and not leaked to the upstream.
```

Comment:
I think having the word "leaked" here makes it sound like this should be default:true.

Perhaps...
```suggestion
StripAuthorizationData ensures that any security tokens used for accessing APIs are stripped and not passed to the upstream.
```